README.md

```diff
@@ -1,3 +1,3 @@
 # textlint-reviewdog-sample
-This istn miss.
+Thsi ints miss.
```

Comment:
🚫 **[textlint]** <eslint.rules.common-misspellings> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This is a commonly misspelled word. Correct it to this (common-misspellings)